tests/tests/control

```diff
@@ -57,7 +61,7 @@ export SYEAR=2021
 export SMONTH=03
 export SDAY=22
 export SHOUR=06
-export RESTART_INTERVAL=12
+export RESTART_INTERVAL="12 -1"
```

Comment:
I didn't read the code carefully, it seems. Does `-1` mean at the end of the forecast run?